### ISSUE TITLE
Add support for homescreen widget?

### ISSUE THREAD

**Issue by e-alfred** [readrops/Readrops#67]
Hello,

your RSS reader is awesome, but it would be great to have a widget that could be placed on the homescreen.

**Comment by Shinokuni** [comment]
Good idea!

**Comment by Happyfeet01** [comment]
That would be nice

**Comment by moshpirit** [comment]
Yes, please! If I don't have the feed in my homescreen I won't read anything 🙈 I really need to force myself haha
